commander/src/bootstrapping/commands/blockchain/hash.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+import * as crypto from 'crypto';
+import { Command, flags as flagParser } from '@oclif/command';
+import { getBlockchainDBPath, getDefaultPath, getFullPath } from '../../../utils/path';
+import { getPid, isApplicationRunning } from '../../../utils/application';
+import { getBlockchainDB } from '../../../utils/db';
+
+export class HashCommand extends Command {
+	static description = 'Generate SHA256 hash from <PATH>.';
+
+	static examples = ['blockchain:hash', 'blockchain:hash --data-path ./data'];
+
+	static flags = {
+		'data-path': flagParser.string({
+			char: 'd',
+			description:
+				'Directory path to specify where node data is stored. Environment variable "LISK_DATA_PATH" can also be used.',
+			env: 'LISK_DATA_PATH',
+		}),
```

Comment:
There is a `data-path` in every other command. Would not it be nice to export such flags independently and reuse. Something like below. 

import commonFlags from '../common_flags`

```
static flags = {
     commonFlags.dataPath
}
```